pom.xml

```diff
@@ -25,7 +25,7 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <maven.compiler.source>1.8</maven.compiler.source>
     <maven.compiler.target>1.8</maven.compiler.target>
-    <maven.version>3.5.4</maven.version>
+    <maven.version>3.6.2</maven.version>
```

Comment:
Maybe it would be good to raise the Maven version even further. The current Maven version is 3.8.6 as far as I know.

Reply:
upgraded to 3.8.6